controllers/tutorial.controller.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-console */
+const { response } = require('express');
+const { Tutorial } = require('../models/tutorial.schema');
+const mangoose = require('../init/db');
+
+// create Document or insert
+const createTutorial = async (req, res) => {
+  try {
+    const demoTutorial = new Tutorial({
+      id: "1",
+      published: true,
+    });
+    const result = await demoTutorial.save();
+    // eslint-disable-next-line no-console
+    console.log(result);
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.log(err);
+  }
+};
+
+const getTutorialById = async (req, res) => {
+  const findId = req.params.id;
+  try {
+    const tutorials = await Tutorial.find({ id: findId });
+    res.json(tutorials);
```

Comment:
1. This API should only return a single Tutorial.
2. If the given tutorial ID doesn't exist in the database then it should return a proper response with `404` response code.